plugins/GymFactory/include/gympp/Metadata.h

```diff
@@ -105,6 +105,9 @@ class gympp::PluginMetadata
     std::string modelFileName;
     std::string worldFileName;
 
+    size_t gazeboUpdateRate;
+    size_t environmentUpdateRate;
```

Comment:
Any specific reason for only supporting integer rates? 0.5 Hz (i.e. one once every two second) seems to be a perfectly valid  update rate, and also the python interface supports floating point frequencies.

Reply:
Yes these should be `double`